www/Themes/Generic-theme/style.css

```
@@ -574,7 +574,8 @@ p.description a, p.description a:visited {
 .host_unreachable, .unknown     {background-color: var(--host-unreachable-unknown-background-color);}
 .host_downtime      {background-color: var(--host-downtime-background-color);}
 
-.service_ok,.host_up, .success  {background-color: var(--service-ok-host-up-success-background-color);}
+.service_ok,.host_up, .success  {background-color: var(--service-ok-host-up-success-background-color);
+color:#fff}
```

I guess there is already a variable for `#fff`
```suggestion
.service_ok,.host_up, .success  {
    background-color: var(--service-ok-host-up-success-background-color);
    color: #fff;
}
```